junitparser/flavors/pytest_xml.py

```diff
@@ -0,0 +1,216 @@
+import itertools
+from junitparser.junitparser import JUnitXml, JUnitXmlError, Result, TestSuite as OrigTestSuite, Attr, \
+    SystemOut, SystemErr, TestCase as OrigTestCase, System
+
+
+class PytestXml(JUnitXml):
```

Comment:
Maybe name it to `JUnitXml` as well. Could result in less cognitive load. Flavors are separated by file names already.

Reply:
Yes, this should be
```suggestion
class JUnitXml(junitparser.JUnitXml):
```
Then user code looks identical, no matter which import they use:
```
from junitparser.junitparser import JUnitXml
from junitparser.flavors.pytest_xml import JUnitXml
```